serrano/resources/preview.py

```diff
@@ -67,11 +68,21 @@ def get(self, request):
         # depend on data to exist!
         header = []
         ordering = OrderedDict(view_node.ordering)
+        concepts = view_node.get_concepts_for_select()
+
+        # Prepare an HTMLExporter
+        exporter = processor.get_exporter(HTMLExporter)
+
+        objects = []
+        header = []
+
+        # Skip pk field.
+        for i, f in enumerate(exporter.header[1:]):
```

Comment:
Good eye. The reason is because [`include_pk`](https://github.com/chop-dbhi/avocado/blob/master/avocado/query/pipeline.py#L29-L30) is used to include the primary key. However, this should not be necessary if I simply set `include_pk` in the `get_exporter` call..